From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Thu, 10 Aug 2023 16:57:47 -0300
Subject: workspaceAnimation: Notify 'progress' property properly

The getter of the MonitorGroup's 'progress' property sets
some values based on it, but doesn't notify the property
change.

Fix that by calling this.notify('progress') when the setter
is called.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2881>
(cherry picked from commit f1db3498ebbf0d4961385685047fcf1a07dd500c)
---
 js/ui/workspaceAnimation.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/js/ui/workspaceAnimation.js b/js/ui/workspaceAnimation.js
index ab405b3..93be07e 100644
--- a/js/ui/workspaceAnimation.js
+++ b/js/ui/workspaceAnimation.js
@@ -214,6 +214,8 @@ const MonitorGroup = GObject.registerClass({
             this._container.x = Math.round(p * this.baseDistance);
         else
             this._container.x = -Math.round(p * this.baseDistance);
+
+        this.notify('progress');
     }
 
     get index() {
